src/lightning_app/utilities/add_meta.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from bs4 import BeautifulSoup
+
+
+@dataclass
+class AppInfo:
```

Comment:
Originally, the idea was to have that in the .lightning file that is stored alongside the app. Today we save only the name of the app there. 

Are we sure about splitting this up?

Reply:
Agree with @awaelchli, we already have `AppConfig` object: https://github.com/Lightning-AI/lightning/blob/7dbd038a9455f3c67da3fe19ef579058feef51b2/src/lightning_app/utilities/packaging/app_config.py#L13 which is the `.lightning` file and can be used to contain configurations or meta data of the app

Reply:
this sort of API requirement came from exec team.